src/librustc_data_structures/indexed_set.rs

```diff
@@ -196,9 +140,7 @@ impl<T: Idx> IdxSet<T> {
 
     pub fn iter(&self) -> Iter<T> {
         Iter {
-            cur: None,
-            iter: self.words().iter().enumerate(),
-            _pd: PhantomData,
+            iter: self.0.iter()
```

Comment:
Is there any point in the wrapper here? Technically it hides possible optimizations (TrustedLen, DoubleEnded, etc) that might be implemented